app/services/audit_results.rb

```diff
@@ -100,15 +100,12 @@ class AuditResults
   ].freeze
```

Comment:
i think you'll also want to get rid of the corresponding `also see HONEYBADGER_REPORT_CODES` in the above list (couldn't attach the comment to that line, since it doesn't show by default in the diff context).

Reply:
cool thanks, repushed